etcd/map.jinja

```diff
@@ -1,6 +1,6 @@
 {% set server = salt['grains.filter_by']({
     'Debian': {
-        'pkgs': ['etcd', 'python-etcd'],
```

Comment:
User should decide whether to use package or anything else. Use traditional ```source: { "engine": "pkg" }``` structure (eg. see salt-formula-aptly).